src/components/show-products/index.js

```diff
@@ -63,7 +63,15 @@ export const ShowProducts = () => {
     }
   };
 
-  const deleteSearchTerm = () => setSearchTerm('');
+  const handleDeleteProduct = (productID) => {
+    if (window.confirm('Are you sure you want to delete?')) {
+      deleteProduct(productID, storedValue)
+        .then(() => window.alert('Successfully deleted product.'))
+        .catch(() =>
+          window.alert('It was not possible to delete the product. Try again.'),
+        );
+    }
+  };
```

Comment:
I love the way you did this feature, it's easy to read and has all the things the issue asks for

Reply:
Loving this :heart: What about prompting user the name of the item they are trying to delete? I think this would be a plus :100:

Reply:
Done!